doc/BoxPlatform.md

```diff
@@ -1,71 +1,133 @@
+            }];
+    [task resume];
+};
 
-@implementation AppDelegate
+// Create a client for the authenticated user; the client is keyed to that user's unique ID for caching purposes
+// The unique ID can be any identifier for the user, or a dummy value
+ServerAuthUser *user = [[ServerAuthUser alloc] initWithUniqueID:@"UNIQUE_ID"];
+BOXContentClient *boxClient = [BOXContentClient clientForServerAuthUser:user
+                                                initialToken:nil
+                                                fetchTokenBlockInfo:nil
+                                                fetchTokenBlock:fetchTokens];
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+// Example of making an API call
+BOXUserRequest *request = [boxClient currentUserRequest];
+[request performRequestWithCompletion:^(BOXUser *user, NSError *error) {
+    if (error) {
+        NSLog(@"Error getting user info");
+    } else {
+        NSLog(@"Retrieved user with login: %@", user.login);
+    }
+}];
+```
 
-    _boxClient = [BOXContentClient clientForServerAuthUniqueId:@"A_UNIQUE_ID"
-                                                  initialToken:nil
-                                           fetchTokenBlockInfo:nil
-                                               fetchTokenBlock:fetchTokenBlock];                                                          
+__Swift:__
+```swift
+let fetchToken: ServerAuthFetchTokenBlock = { (
+        uniqueID: String?,
+        fetchTokenBlockInfo: [AnyHashable: Any]?,
+        completion: ((String?, Date?, Error?) -> Void)!
+    ) -> Void in
     
-    //Example of making an API call
-    BOXUserRequest *request = [_boxClient currentUserRequest];
-    [request performRequestWithCompletion:^(BOXUser *user, NSError *error) {
-        if (error){
-            NSLog(@"Error getting user info");
+    // NOTE: This code is specific to your app; it is responsible for retrieving new tokens from a secure server
+    NSLog("Attempting to retrieve new access token from server for unique ID: \(uniqueID)");
+    
+    // Make a request to the secure sever for a token
+    let url = URL(string: "https://YOUR_SECURE_SERVER.example.com/token?id=\(uniqueID ?? "")")!
+    let task = URLSession.shared.dataTask(with: url) { data, response, error in
+        
+        if error != nil {
+            NSLog("Failed to retrieve new access token from server")
+            completion(nil, nil, error)
+            return
         }
-        else {
-            NSLog(@"Retrieved app user info. Login=%@", user.login);
+        
+        guard let httpResponse = response as? HTTPURLResponse, httpResponse.statusCode == 200 else {
+            NSLog("Server returned incorrect response")
+            completion(nil, nil, Error("Token sever returned incorrect response"))
+            return
         }
-    }];
-
-    return YES;
+        
+        let token = String(data: data!, encoding: .utf8)!
```

Comment:
I suppose this is fine for sample code, but these `!` are invitations to crash, and I wouldn't do it in production.